server/spec/mutations/create_artist_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Artist, type: :request do
+  describe 'create_artist_mutation' do
+    let(:query_string) {
+      <<-GraphQL
+        mutation ($name: String!) {
+          createArtist(input: { name: $name }) {
+            artist {
+              id
+              name
+            }
+          }
+        }
+      GraphQL
+    }
+    let(:result) { ServerSchema.execute(query_string, variables: { name: name }) }
+    let(:result_data) { result['data']['createArtist'] }
+
+    context 'when artist name is valid' do
+      let(:name) { 'artist' }
+
+      it 'should create an artist' do
+        expect(result_data['artist']['name']).to eq 'artist' and change(Artist, :count).by(1)
+      end
+    end
+
+    context 'when same artist name already exists' do
+      before do
+        create(:artist, name: 'artist')
+      end
```

Comment:
複数の文がなければ，ブロックで書くことが多いかな。このままでも大丈夫。
```suggestion
      before { create(:artist, name: 'artist') }
```